fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/dashboard/CardsRestClient.kt

```diff
@@ -163,6 +173,40 @@ class CardsRestClient @Inject constructor(
         )
     }
 
+    data class DynamicCardResponse(
+        @SerializedName("id") val id: String,
+        @SerializedName("title") val title: String,
+        @SerializedName("remote_feature_flag") val remoteFeatureFlag: String,
+        @SerializedName("featured_image") val featuredImage: String,
+        @SerializedName("url") val url: String,
+        @SerializedName("action") val action: String,
+        @SerializedName("order") val order: String,
+        @SerializedName("rows") val rows: List<DynamicCardRowResponse>
```

Comment:
According to the spec (pc8HXX-1nA-p2) we expect some of the properties are nullable:
```suggestion
        @SerializedName("id") val id: String,
        @SerializedName("title") val title: String?,
        @SerializedName("remote_feature_flag") val remoteFeatureFlag: String?,
        @SerializedName("featured_image") val featuredImage: String?,
        @SerializedName("url") val url: String?,
        @SerializedName("action") val action: String?,
        @SerializedName("order") val order: String,
        @SerializedName("rows") val rows: List<DynamicCardRowResponse>
```

Reply:
Great catch @justtwago 👍 
I accepted the suggestion and changed the CardModel fields to be nullable too with 6a1d32ef84faf54246f5061c093956150e3f5ae9